zebrad/src/application.rs

```diff
@@ -135,10 +136,28 @@ impl Application for ZebradApp {
 
 impl ZebradApp {
     fn level(&self, command: &EntryPoint<ZebradCmd>) -> String {
-        if let Ok(level) = std::env::var("ZEBRAD_LOG") {
-            level
-        } else if command.verbose {
+        use ZebradCmd::*;
+        // Tracing sends output to stdout, so we disable it for some commands.
+        //
+        // TODO: send tracing output to stderr. This change requires an abscissa
+        //       update, because `abscissa_core::component::Tracing` uses
+        //       `tracing_subscriber::fmt::Formatter`, which has `Stdout` as a
+        //       type parameter. We need `MakeWriter` or a similar type.
+        let command_uses_stdout = match command.command {
+            Some(Generate(_)) | Some(Help(_)) | Some(Revhex(_)) | Some(Version(_)) => true,
+            _ => false,
+        };
```

Comment:
TIOLI: I think this would be slightly cleaner as a `Command::uses_stdout()` method.

Reply:
Fixed in a9c03a8b8e3d8458c679e08f2ff935f7ed8d2436